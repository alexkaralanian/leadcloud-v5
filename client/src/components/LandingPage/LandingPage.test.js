describe("Landing Page Component", () => {
  test("", () => {});
});
