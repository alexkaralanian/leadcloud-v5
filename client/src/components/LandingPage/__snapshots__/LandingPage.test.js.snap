// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landing Page Component renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <LandingPage />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="LandingPage"
>
        <div
                className="ContentBox"
        >
                <div
                        className="PrimaryHeading"
                >
                        <h1
                                className="PrimaryHeading_Main"
                        >
                                Tempo
                        </h1>
                        <h2
                                className="PrimaryHeading_Sub"
                        >
                                by LeadCloud
                        </h2>
                </div>
                <GoogleButton />
        </div>
</div>,
      "className": "LandingPageContainer",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="ContentBox"
>
          <div
                    className="PrimaryHeading"
          >
                    <h1
                              className="PrimaryHeading_Main"
                    >
                              Tempo
                    </h1>
                    <h2
                              className="PrimaryHeading_Sub"
                    >
                              by LeadCloud
                    </h2>
          </div>
          <GoogleButton />
</div>,
        "className": "LandingPage",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div
              className="PrimaryHeading"
>
              <h1
                            className="PrimaryHeading_Main"
              >
                            Tempo
              </h1>
              <h2
                            className="PrimaryHeading_Sub"
              >
                            by LeadCloud
              </h2>
</div>,
            <GoogleButton />,
          ],
          "className": "ContentBox",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <h1
                  className="PrimaryHeading_Main"
>
                  Tempo
</h1>,
                <h2
                  className="PrimaryHeading_Sub"
>
                  by LeadCloud
</h2>,
              ],
              "className": "PrimaryHeading",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Tempo",
                  "className": "PrimaryHeading_Main",
                },
                "ref": null,
                "rendered": "Tempo",
                "type": "h1",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "by LeadCloud",
                  "className": "PrimaryHeading_Sub",
                },
                "ref": null,
                "rendered": "by LeadCloud",
                "type": "h2",
              },
            ],
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
        ],
        "type": "div",
      },
      "type": "div",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="LandingPage"
>
          <div
                    className="ContentBox"
          >
                    <div
                              className="PrimaryHeading"
                    >
                              <h1
                                        className="PrimaryHeading_Main"
                              >
                                        Tempo
                              </h1>
                              <h2
                                        className="PrimaryHeading_Sub"
                              >
                                        by LeadCloud
                              </h2>
                    </div>
                    <GoogleButton />
          </div>
</div>,
        "className": "LandingPageContainer",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <div
            className="ContentBox"
>
            <div
                        className="PrimaryHeading"
            >
                        <h1
                                    className="PrimaryHeading_Main"
                        >
                                    Tempo
                        </h1>
                        <h2
                                    className="PrimaryHeading_Sub"
                        >
                                    by LeadCloud
                        </h2>
            </div>
            <GoogleButton />
</div>,
          "className": "LandingPage",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <div
                className="PrimaryHeading"
>
                <h1
                                className="PrimaryHeading_Main"
                >
                                Tempo
                </h1>
                <h2
                                className="PrimaryHeading_Sub"
                >
                                by LeadCloud
                </h2>
</div>,
              <GoogleButton />,
            ],
            "className": "ContentBox",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <h1
                    className="PrimaryHeading_Main"
>
                    Tempo
</h1>,
                  <h2
                    className="PrimaryHeading_Sub"
>
                    by LeadCloud
</h2>,
                ],
                "className": "PrimaryHeading",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Tempo",
                    "className": "PrimaryHeading_Main",
                  },
                  "ref": null,
                  "rendered": "Tempo",
                  "type": "h1",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "by LeadCloud",
                    "className": "PrimaryHeading_Sub",
                  },
                  "ref": null,
                  "rendered": "by LeadCloud",
                  "type": "h2",
                },
              ],
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
          ],
          "type": "div",
        },
        "type": "div",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
